scripts/packaging/generic_distro.py

```diff
@@ -185,26 +185,9 @@ def add_file(self, file_path):
             self.package_files.append(os.path.relpath(world_project_file, self.webots_home))
 
         if file_path.endswith('.proto') and str(os.path.sep + 'protos' + os.path.sep) in file_path:
-            # compute MD5 fo PROTO file
             if not os.path.exists(absolute_path):
                 print_error_message_and_exit(f"Missing file: {absolute_path}")
 
-            md5Value = self.compute_md5_of_file(absolute_path)
```

Comment:
Since it no longer appears to be necessary, I think we can also clean up the `compute_md5_of_file` function